contracts/Cross-chain/OmnichainGovernanceExecutor.sol

```diff
@@ -214,13 +214,13 @@ contract OmnichainGovernanceExecutor is ReentrancyGuard, BaseOmnichainController
             uint256[] memory values,
             string[] memory signatures,
             bytes[] memory calldatas,
+            uint256 pId_,
```

Comment:
I don't think the proposal ID is included in the payload. The proposal payload is sent [here](https://github.com/VenusProtocol/governance-contracts/pull/21/files#diff-448b5741452de8d78241b8840f340010400a0635c9b66e7d3ec8eae2e37a41a9R135). The proposal ID is included in the first level of the message

Reply:
It is included in the payload as far as I could tell. Without it the tests fail because the proposal Id was being read as the proposalType

Reply:
I see. It must be passing now because the last argument is defaulting to zero